ktor-shared/ktor-shared-serialization-gson/build.gradle.kts

```diff
@@ -6,6 +6,7 @@ val kotlin_version: String by project.extra
 kotlin.sourceSets {
     jvmMain {
         dependencies {
+            api(project(":ktor-server"))
```

Comment:
Can you explain why do you need this dependency here? If you keep it, `ktor-client` will have transitive dependency on it too.

Reply:
I need BadRequestException which is placed in io.ktor.server.plugin. So, I've added ktor-client. Should I use another exception?

Reply:
I fixed it with throwing custom ContentConvertException from converters and catching them in ContentNegotiation. I've deleted ktor-client dependency.